TeachingRecordSystem/src/TeachingRecordSystem.Core/Services/WorkforceData/BlobStorageTpsExtractStorageService.cs

```diff
@@ -18,6 +19,14 @@ public async Task<string[]> GetPendingImportFileNames(CancellationToken cancella
         return fileNames.ToArray();
     }
 
+    public async Task<string?> GetPendingEstablishmentImportFileName(CancellationToken cancellationToken)
+    {
+        var blobContainerClient = blobServiceClient.GetBlobContainerClient(TpsExtractsContainerName);
+        var fileNames = new List<string>();
+        await GetFileNamesAsync(blobContainerClient, EstablishmentsFolderName, true, fileNames, cancellationToken);
```

Comment:
Why doesn't `GetFileNamesAsync` return the file names instead of appending to a list? (I missed this before.) Could probably be just `GetFileNames` too.